common.blocks/jquery/__event/_type/jquery__event_type_pointernative.js

```diff
@@ -532,7 +532,7 @@ var touchEvents = {
     removePrimaryPointer : function(pointer) {
         if(pointer.isPrimary) {
             this.firstTouch = null;
-            //this.firstXY = null;
+            // this.firstXY = null; // no reason to left comments with code like this without a note
```

Comment:
Can you make a comment here, like this:

~~~js
// TODO: It seems that, flushing `firstXY` flag explicitly in `touchmove` handler is enough.
// Original code from polymer doing this on every `removePrimaryPointer`, but looks 
// like it is harmful in some of our usecases. 
~~~